numpy/_build_utils/src/apple_sgemv_fix.c

```diff
@@ -66,12 +69,27 @@ static int AVX_and_10_9 = 0;
 /* Dynamic check for AVX support
  * __builtin_cpu_supports("avx") is available in gcc 4.8,
  * but clang and icc do not currently support it. */
-#define cpu_supports_avx()\
-(system("sysctl -n machdep.cpu.features | grep -q AVX") == 0)
-
+static inline int cpu_supports_avx() {
+    int enabled, r;
+    size_t length = sizeof(enabled);
+    r = sysctlbyname("hw.optional.avx1_0", &enabled, &length, NULL, 0);
+    if ( r == 0 && enabled != 0) 
+        return 1;
+    else
+        return 0;
+}
+	
 /* Check if we are using MacOS X version 10.9 */
-#define using_mavericks()\
-(system("sw_vers -productVersion | grep -q 10\\.9\\.") == 0)
+static inline int using_mavericks() {
```

Comment:
Like so:
```
static inline int
using_mavericks(void)
{
```
also above.

Reply:
Strictly speaking, it should be `NPY_INLINE`, but now that we are requiring C99, `inline` is probably OK.